### DISCUSSION TITLE
Custom text push notifications/emails

### DISCUSSION THREAD

**Discussion by camel-nz** [sct/overseerr#2079] [Feature Requests]
Hi, 
I would love the ability to send out custom text push notifications/emails from the platform so I can make announcements and outage notifications. 

I have a lot of people who use the system i want to be able to notify them when i am taking it down for maintenance. 
No one who uses my server have access to Discord

who would have thought that i need to have change control for my home lab!

**Comment by aunefyren** [comment]
Would love this as well! My users are only used to recieving notifications from Overseerr, so being able to keep the format and such for custom notifications would be great.

**Comment by anonyme123** [comment]
I would love this feature! For me, Id' like to have the request owner displayed in the header of the notification, rather than the body.

**Comment by michaelmurfy** [comment]
Just bumping this as I think it'll be a great addition. Ombi has this feature and I used it to inform users of upcoming server maintenance. It'll be awesome if Overseerr had this feature also.

**Comment by Thunder7ga** [comment]
Concur with this one....I switched from OMBI to this, but the only thing I really miss is the "Mass E-Mail" function to send info/notifications to all the users at once.  I used it several times a year.

**Comment by jarzebowsky** [comment]
Looking for that feature couple of months and found - nothing. Looking forward to this!
